packages/core/admin/admin/src/content-manager/components/BlocksEditor/Wrapper.js

```diff
@@ -0,0 +1,137 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
```

Comment:
how come a. this isn't using `Box` and b. this has all the contained styles for the elements inside? 🤔

Reply:
b. yes as global style overrides all elements, I referred this from wysiwig editor. Please suggest if you have other solution.

Reply:
Well if we're supplying components in `BlocksEditor/index.js` can we not just just the Typography component? Or just use some styled components there instead? I wouldn't just keep doing what we're doing, instead try to think how you'd approach it from scratch.

Reply:
I agree with @joshuaellis, one of the goals of block-based editors is to be able to directly plug your design system to render each block, and to avoid having to write CSS for a big chunk of generated HTML

Reply:
I have updated elements with typography and styled components, please let me know if I missed any existing component from DS